LondonBritishLibrary/orient/BLorient13264.xml

```diff
@@ -0,0 +1,327 @@
+                            <title ref="LIT4872MagdemaWangel">Introduction to the Gospels</title>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ንቅድም፡ በረድኤተ፡ እግዚአብሔር፡ ሎቱ፡ ስብሐት፡ 
+                                <supplied reason="omitted" resp="Strelcyn">ይትባረክ፡ ስሙ፡</supplied> ወይትለዓል፡ ዝክሩ፡ ወፍጽምናሁ፡ ዘእምኔሁ፡ ሢመተ፡ ኵሉ፡ 
+                                ኃይል፡ ወቦቱ፡ ተፍጻሜተ፡ ኵሉ፡ ስእለት፤ ወኀቤሁ፡ ምብጻሐ፡ ኵሉ፡ አስተብቍዖት፡ በጽሒፈ፡ መቅድመ፡ ፬፡ ወንጌላት፡ 
+                                <supplied reason="undefined" resp="Strelcyn">ስቡሓት፡ ወ፡</supplied>ቅዱሳት። ወ፲አቅማራት፡ በሰላመ፡ እግዚአብሔር፡ አሜን። </incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i6">
+                            <locus from="134ra" to="148ra"/>
+                            <title ref="LIT6468Astebeqweot"/>
+                            <note>Prayers of St Simeon Stylites for the days of the week, beginning with Monday.</note>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ንወጥን፡ በረድኤተ፡ እግዚአብሔር፡ ክቡር፡ ወልዑል፡ ወበሥነ፡ ስርሐቱ፡ በጽሒፈ፡ ጸሎት፡ 
+                                ወአስተብቍዖት፡ እንተ፡ ዘስምዖን፡ ዓምዳዊ። ቀዳሚሃ፡ ዘእግዝእትነ፡ ማርያም፡ ወድንግልት፡ በክልኤ።</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i7">
+                            <locus from="148ra" to="148vb"/>
+                            <title>Unidentified Prayer ascribed to St Simeon Stylites</title>
+                            <incipit xml:lang="gez">ካልእ፡ ጸሎቱ፡ ዘቅዱስ፡ አባ፡ ስምዖን፡ በዓለ፡ አምድ፡ ሀበኒ፡ ኦእግዚአብሔር፡ ጸሎተ፡ ዘእንበለ፡ ነጎጋው።</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i8">
+                            <locus from="148vb" to="149va"/>
+                            <title ref="LIT4239Prayer"/>
+                            <incipit xml:lang="gez">ጸሎቱ፡ ዘቅዱስ፡ አባ፡ ኤፍሬም፡ <supplied reason="omitted" resp="Strelcyn">ሶርያዊ፡</supplied> 
+                                ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶርስ፡ ተዘከርኩ፡ ይእተ፡ ሰዓተ፤ ወፈራህኩ። </incipit>
+                        </msItem>     
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">150</measure>
+                                    <measure unit="leaf" type="blank" corresp="#2v #66v #81v #142v #150">5</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>240</height>
+                                        <width>190</width>
+                                    </dimensions>
+                                </extent>
+                                <foliation>Seventeen ouf of eighteen quires are numbered with quiremarks.</foliation>
```

Comment:
quire numbering is not part of `foliation `but of `collation`

according to the TEI guidelines the best element could be `signatures `https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-signatures.html (inside `collation`)